src/cache/main.go

```diff
@@ -121,8 +121,11 @@ func main() {
 			return
 		}
 
-		// Fetch TTL from config file and convert it into a time.Duration in seconds
-		ttl := time.Duration(int(config["ttl"].(float64))) * time.Second
+		ttl := requestBody.ttl
+		if ttl == "" {
```

Comment:
```suggestion
		if ttl == nil || ttl == "" {
```

I haven't used GoLang a whole lot personally. Do we possibly need another condition in here if the value of ttl is nil? Perhaps it should go first?

Reply:
I based this check on the previous checks for the other values.